<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AAT</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../blog/AAT/design/export512px.png</normaloff>../../blog/AAT/design/export512px.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>531</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>현재 시간</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>471</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>531</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>장 오픈 / 마감 여부</string>
    </property>
    <widget class="QRadioButton" name="radioButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>130</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>개장</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>130</width>
       <height>22</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>폐장</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>3</width>
       <height>61</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>40</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>개장 시간이 아닙니다.</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>290</y>
      <width>531</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>서버 연결상태</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>서버 미연결 상태 😅</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>151</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>서버 접속</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>90</y>
       <width>151</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>서버 접속 해제</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>41</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>design/iconmonstr-help-6-240.png</normaloff>design/iconmonstr-help-6-240.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>매매</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_7"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>계좌관리</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>알림설정</string>
    </property>
    <addaction name="actionPC"/>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>데이터</string>
    </property>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>암호화 설정</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>자동실행 설정</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_7">
    <property name="title">
     <string>데이터시각화</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_6"/>
   <addaction name="menu_7"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>자동매매</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>수동매매</string>
   </property>
  </action>
  <action name="actionPC">
   <property name="text">
    <string>PC</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>텔레그램</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>슬랙</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>금융감독원</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>재무재표</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>거래내역</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
